server/repository/errors.go

```diff
@@ -0,0 +1,29 @@
+package repository
+
+import "errors"
+
+const (
```

Comment:
ここ、constantじゃないからvarにしないと落ちる(私のガバ
```suggestion
var (
```